### PR TITLE
Move app-bar to main panel

### PR DESCRIPTION
**Pull request by timja**
Requires https://github.com/jenkinsci/jenkins/pull/7051

App-bar is getting a new feature to allow it to be in the sidepanel, it's now required to place it in the section you want it displayed.

_Note: no compatible way of doing this currently, it needs to be released when core is released, given there's only 3 plugins using it this should be fine_

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -50,7 +50,7 @@
 
     <properties>
         <changelist>999999-SNAPSHOT</changelist>
-        <jenkins.version>2.346.1</jenkins.version>
+        <jenkins.version>2.367-rc32834.83151ee78b_12</jenkins.version>
```
Comment (timja) [inline pom.xml:53]: requires https://github.com/jenkinsci/jenkins/pull/7051

**Comment by NotMyFault** [comment]
I'll take a look next week (same for the core PR), but thanks for the PR so far 👏🏻

**Review by NotMyFault — APPROVED** [review]
Tested with the core PR, thanks a bunch 👍🏻

File: pom.xml (hunk 1 above)
Comment (NotMyFault) [inline pom.xml:53]:
```suggestion
        <jenkins.version>2.374</jenkins.version>
```

**Review by NotMyFault — APPROVED** [review]

**Pull request merged**